_data/maps-collection-metadata.csv

```diff
@@ -1,76 +1,99 @@
+coll070,coll070.jpg,"Transit Routes in Vancouver, 1953",British Columbia Electric.,1953,,,"14 maps on one sheet of Downtown Vancouver transit routes, 1953",scale not given,"Vancouver, BC",49.2827,-123.1207,"Koerner Library Map Collection, University of British Columbia",624.9gmbm Vancouver 1953 Bt,image/jpeg,eng,https://transitmap.net/vancouver-1953/,http://resolve.library.ubc.ca/cgi-bin/catsearch?bid=2803620,,,
+coll071,coll071.jpg,"Vancouver Land Use, 1880s",Need to check,1880,,,"A map of Vancouver land use, 1880s"," 1:50,000","Vancouver, BC",49.2827,-123.1207,"Koerner Library Map Collection, University of British Columbia",624.9p Vancouver 1880s,image/jpeg,eng,https://searcharchives.vancouver.ca/vancouver-in-1880s,None,,,
+coll072,coll072.jpg,"First Peoples Vancouver, 1850s",Need to check,1850,,,"A map of First Peoples' Vancouver, 1850s"," 1:50,000","Vancouver, BC",49.2827,-123.1207,"Koerner Library Map Collection, University of British Columbia",624.9p Vancouver 1880s V,image/jpeg,eng,https://searcharchives.vancouver.ca/first-nations-vancouver-in-1850s,None,,,
+coll073,coll073.jpg,"Greater Vancouver Traffic Noise, 1971",Vancouver Sun.,1971,,,"A map as part of a newspaper clipping, showing traffic noise levels of municipalities in Greater Vancouver, 1971",scale not given,"Greater Vancouver, BC",49.23285,-123.024295,"Koerner Library Map Collection, University of British Columbia",624.9gmag Vancouver 1971 V,image/jpeg,eng,,http://resolve.library.ubc.ca/cgi-bin/catsearch?bid=2803614,,,
+coll074,coll074.jpg,"Greater Vancouver Traffic Volumes, 1967",Greater Vancouver Regional District. Planning Department.,1967,1967-1968,,"A map of Greater Vancouver showing summer travel times at 4:30 by fastest travel routes, 1967-68"," 1:190,080","Greater Vancouver, BC",49.23285,-123.024295,"Koerner Library Map Collection, University of British Columbia",624.9gmag Greater Vancouver 1967 G,image/jpeg,eng,,http://resolve.library.ubc.ca/cgi-bin/catsearch?bid=2803701,,,
+coll075,coll075.jpg,"Maps of the University of British Columbia, 1928-2000",University of British Columbia.,1928,1928-2000,,"A variety of maps of the University of British Columbia, 1928-2000",scales not given,"Vancouver, BC",49.2827,-123.1207,"Koerner Library Map Collection, University of British Columbia",624.9a,image/jpeg,eng,,None,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,,,,,,,,,,,,,,,,,,,
+,,https://searcharchives.vancouver.ca/kitsilano-east-half-building-outlines,,,,,,,,,,,,,,,,,,
```

Comment:
Hi @cfokkens, I can't figure out why this url is here. Looks like maybe a cell in the Google sheet got away?

Reply:
Can you verify this is intentional, in which case I'll merge the request. Otherwise, go ahead and delete this first. Thanks!

Reply:
Good catch - I think I was copying and pasting some links and probably forgot to delete that one. I'll update this now.